src/Symfony/Bundle/SecurityBundle/Tests/Functional/app/SecurityHelper/config.yml

```diff
@@ -7,6 +7,10 @@ services:
         alias: security.helper
         public: true
 
+    security.token_storage.alias:
```

Comment:
test.security.token_storage

Reply:
changed to functional.test.security.token_storage to be consistent with the alias name above.